man/fi_verbs.7.md

```diff
@@ -107,6 +113,48 @@ The verbs provider supports a subset of OFI features.
 
   * Not thread safe.
 
+#### Fork
+: The support for fork in the provider has the following limitations:
+  * Fabric resources like endpoint, CQ, EQ, etc. should not be used in the
+    forked process.
+  * The memory registered using fi_mr_reg has to be page aligned since ibv_reg_mr
+    marks the entire page that a memory region belongs to as not to be re-mapped
+    when the process is forked (MADV_DONTFORK).
+
+# RUNTIME PARAMETERS
+
+The verbs provider checks for the following environment variables.
+
+#### Variables specific to RDM endpoints
+
+##### FI_VERBS_IFACE
+: The prefix or the full name of the network interface associated with the IB device (default: ib)
```

Comment:
Probably not. When there are multiple IB devices of same type, their domain names would be the same.